src/Core/Collections/Leaderboard/config/leaderboardItems.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import LeaderboardType from 'src/backend/shared/constants/LeaderboardType';
+import * as Icons from '../../iconography';
+import { LeaderboardItem } from '../components/LeaderboardItem';
+
+const { RECORD_EXAMPLE_AUDIO, IGBO_DEFINITION, VERIFY_EXAMPLE_AUDIO } = LeaderboardType;
+
+const titleMap = {
+  [VERIFY_EXAMPLE_AUDIO]: 'Verified recorded sentence',
+  [RECORD_EXAMPLE_AUDIO]: 'Recorded example sentence',
+  [IGBO_DEFINITION]: 'Referral champions',
+};
+
+export const leaderboardItems = [
+  {
+    children: (
+      <LeaderboardItem Icon={<Icons.Done stroke="rgba(189, 0, 255, 1)" />} title={titleMap[VERIFY_EXAMPLE_AUDIO]} />
+    ),
```

Comment:
it feels a bit odd to have functional components being called within object, instead you can just have the props that would be passed into the `LeaderboardItem` component as fields in this object (this way you can remove the `children` object

```suggestion
    icon: Done,
    title: titleMap[VERIFY_EXAMPLE_AUDIO],
```
let me know if you have any questions

Reply:
This implementation satisfies the `O` in `SOLID` principle moreover it makes the component very flexible. Material UI adopts this composition type architecture for their framework.

Reply:
sounds good to me!